districtm-dmx/districtm-dmx-htb.js

```diff
@@ -154,7 +154,7 @@ function DistrictmDmxHtb(configs) {
             obj.id = System.generateUniqueId();
             obj.tagid = String(tag.xSlotRef.dmxid);
             tag.xSlotRef.id = obj.id;
-            obj.secure = window.location.protocol === 'https' ? 1 : 0;
+            obj.secure = 1;
```

Comment:
Just to confirm, the intent is to force the use of secure protocol in all scenarios?

Reply:
Yes, we have the same change pending for prebid 3.0